functions/src/scripts/supabase-import.ts

```diff
@@ -107,6 +107,14 @@ async function importDatabase(kinds?: string[]) {
 
   if (shouldImport('user'))
     await importCollection(client, firestore.collection('users'), 'user', 500)
+  if (shouldImport('portfolioHistory'))
```

Comment:
By convention this would be named `userPortfolioHistory` but I don't care that much about the convention.

Reply:
Changed it to match, thanks!